spec/services/time_ago_spec.rb

```diff
@@ -0,0 +1,86 @@
+describe TimeAgo do
```

Comment:
what is that case? this is the current implementation
```
if time_with_zone < 1.week.ago
        I18n.l(time_with_zone.to_date, format: :long)
      else
        I18n.t('common.time_ago', date: time_ago_in_words(time_with_zone))
      end
```
so if it is less than a week ago, you'll get the relative time, otherwise you'll get just the date (no time). In the previous implementation with the fork, you could get something like "Sunday at 3:54 pm" but the Rails i18n gem doesn't have that same kind of day-of-the-week formatting (we could write it if it's really important). are there places where I've replaced the implementation that are just time data? cuz then yeah we would need to change that. thanks!